hoomd/pytest/test_snapshot.py

```diff
@@ -125,6 +125,104 @@ def test_configuration(s):
         assert s.configuration.dimensions == 3
 
 
+def test_wrap(s):
+
+    def generateOutside(box, inside, multipliers):
```

Comment:
Python typically prefers snake case names.
```suggestion
    def generate_outside(box, inside, multipliers):
```